plugins/coral-plugin-love/client/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Coral Plugin Love",
+  "slot": "Comment.Detail"
```

Comment:
I'm not sure about this naming convention. We probably need to talk with everyone and agree on a naming scheme. I personally prefer something like `comment.detail` and `comment.infoBar`.